### DISCUSSION TITLE
How to cancel filepicker?

### DISCUSSION THREAD

**Discussion by hongquan** [zellij-org/zellij#3527] [Q&A] [answered]
I binded "Alt F" to opern filepicker. But after filepicker opens and I don't want to pick a file, just want to cancel, then how to do it?

I pressed "Esc" but nothing happened.

**Comment by cristiand391** [comment, marked as answer]
Last time I looked at the code I didn't find a defined key comb to "cancel", I just kill the panel (most of the time it's open in a new floating pane)
